scripts/new-chain.ts

```diff
@@ -0,0 +1,187 @@
+import fs from 'fs'
+import path from 'path'
+import { ethers } from 'ethers'
+import dotenv from "dotenv";
+import { ScriptError, ScriptErrorCode, runScript } from './utils';
+import { ADDRESS, CODEHASH, FACTORY_BYTECODE, SIGNER } from './constants';
+
+dotenv.config()
+
+async function newChainWrapper() {
+	const summary: Record<string, unknown> = {};
+	summary.SUCCESS = false
+	summary.LABEL_OPERATION = "--remove-label"
```

Comment:
nit: Why SCREAMING_SNAKE_CASE? I would just use camelCase to make it more JSON-y

Reply:
Updated 👍🏾